src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry.PowerShell
+{
+    public class InstanceCloseEvent : TelemetryEvent
+    {
+        public InstanceCloseEvent(
+            int PmcExecuteCommandCount,
```

Comment:
Parameter names are usually `camelCase`. We don't seem to have an explicit rule in https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md, but we I will add it.

Reply:
More than 1 file has this issue, please address them all.

Reply:
@nkolev92 
Addressed the issue. 
I was receiving comments on properties about Pascal casing then I used replace all to address it, then this is byproduct of it.